src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -567,7 +567,7 @@ public override bool IsReferenceType
         {
             get
             {
-                return _underlyingType.IsReferenceType;
+                return _underlyingType.IsErrorType() ? false : _underlyingType.IsReferenceType;
```

Comment:
Yes. Actually almost all the IDE tests hit this because the code doesn't include code or references for `ValueTuple`. I was wondering whether this should just always return `false` assuming we also add a check that the underlying type is a struct. Is there any reason for the tuple to be a reference type?